README.md

```diff
@@ -86,9 +86,9 @@ For the unsupervised training, we are using a related problem for which we alrea
 
     ```
     -network                  To select which network architecture to use
-                                                       
+
```

Comment:
These whitespace changes are unrelated, could you remove them from the patch?

Reply:
i have no idea why this keeps happening. I didn't touch this part of the file at all. i tried to revise this but this still keep popping up

Reply:
It's probably your text editor doing this automatically. It's OK, let's ignore it.